applications/FluidDynamicsApplication/python_scripts/compressible_slip_wall_process.py

```diff
@@ -93,6 +94,13 @@ def Check(self):
             msg = "Parameter rampup_period must greater than the interval ({0} < {1}).".format(self.rampup_period, full_duration)
             raise ValueError(msg)
 
+        self.model_part.HasNodalSolutionStepVariable(self.variable)
```

Comment:
Shouldn't we do something with this?

Reply:
This is what happens when you take a coffee break mid-coding :laughing:.

Fixed.